docs/tre-admins/upgrading-resources.md

```diff
@@ -0,0 +1,57 @@
+<!-- markdownlint-disable-file MD046 -->
+# Upgrading Resources Version
+
+Azure TRE workspaces, workspace services, workspace shared services, and user resources are [Porter](https://porter.sh/) bundles. Porter bundles are based on [Cloud Native Application Bundles (CNAB)](https://cnab.io/).
+
+When a new bundle version becomes avaialble, users can upgrade their resources to a newer version after building, publishing and registering the bundle template.
+
+Upgrades (and downgrades) are based on [CNAB bundle upgrade action](https://getporter.org/bundle/manifest/#bundle-actions).
+
+Bundle template versions follow [semantic versioning rules](../tre-workspace-authors/authoring-workspace-templates.md#versioning), only minor and build version upgrades are allowed by default with Azure TRE upgrade mechanism, major versions upgrades and any version downgrades are blocked.
+
+!!! Warning
+    We do not recomend upgrading automatically a major version as those changes might include breaking changes, potential data loss or cause service unavaialbility.
```

Comment:
Suggest remove this sentence and, instead, use the text I suggested in the sentence above. This then becomes a note on how to do a major version upgrade with force.